src/markov/markov_approx.jl

```diff
@@ -130,3 +130,50 @@ function _rouwenhorst(p::Real, q::Real, m::Real, Δ::Real, n::Integer)
         return linspace(m-Δ, m+Δ, n), θN
     end
 end
+
+
+"""
+Accepts the simulation of a discrete state Markov chain and estimates
+the transition probabilities via maximum likelihood.
```

Comment:
> (Julia docs don't support mathjax/latex, right?)

It should: https://juliadocs.github.io/Documenter.jl/stable/man/latex/

@sglyon could we add support for this?